apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -4,15 +4,19 @@ import {SET_SECTION} from '@cdo/apps/redux/sectionDataRedux';
  // TODO(caleybrock): define a shape for sectionAssessment data that gets stored in redux.
 const initialState = {
   assessmentsByScript: {},
+  assessmentsStructureByScript: {},
```

Comment:
What's the difference between `assessmentsByScript` and `assessmentsStructureByScript`? Adding a PropTypes shape for these (as mentioned in the TODO) could be a good way to document their structure, but ok to not block this PR on that

Reply:
One is the student assessment responses and one is the assessments structure. Will document more in the future.